### PR TITLE
Content placeholders

### PR DESCRIPTION
**Pull request by bankent1**
Added the logo to the banner and some placeholders for the left and right content sections. It looks like webpack updated too so might want to test that.

### CODE REVIEW

**Comment by bankent1** [comment]
@vinamartin @Kjames5269 @jhunzik

File: src/index.js [hunk 1]
```diff
@@ -36,43 +68,52 @@ const BannerGrid = styled.div`
...
-        <p>-=- {text} -=-</p>
-    </div>
+  <div>
+    <p>-=- {text} -=-</p>
+  </div>
 )
 
 const Logo = () => {
-    return <img src={logo} alt="Logo" height="100px"/>;
+  return <img src={logo} alt="Logo" height="75px" />;
```
Comment (vinamartin) [inline src/index.js]: ✏️ This is a slight nitpick, I think the logo might be able to be larger to fill up the space better. Right now I think it's a bit too small.

File: src/index.js [hunk 2]
```diff
@@ -36,43 +68,52 @@ const BannerGrid = styled.div`
...
-          <div>EST: <Clock format={'HH:mm'} ticking={true} timezone={'US/Eastern'} /></div>
-        </ClockGridVert>
-        <ClockGridVert>
-          <div>CST: <Clock format={'HH:mm'} ticking={true} timezone={'US/Central'} /></div>
-          <div>ACT: <Clock format={'HH:mm'} ticking={true} timezone={'Australia/Sydney'} /></div>
-        </ClockGridVert>
-      </ClockGridHorz>
+        <ClockGridHorz>
+          <ClockGridVert>
+            <div>PST: <Clock format={'HH:mm'} ticking={true} timezone={'US/Pacific'} /></div>
```
Comment (jhunzik) [inline src/index.js]: ❓ Should this be MST?

Comment (bankent1) [reply src/index.js]: Yeah I don't actually know the 4(?) timezones that we are going to use so they are just placeholders for now.

**Pull request merged**